rmw_connext_cpp/src/functions.cpp

```diff
@@ -29,10 +33,21 @@
 #include <rmw/error_handling.h>
 #include <rmw/types.h>
 
+#include <rmw/impl/cpp/macros.hpp>
+
 #include "rosidl_typesupport_connext_cpp/identifier.hpp"
 #include <rosidl_typesupport_connext_cpp/message_type_support.h>
 #include <rosidl_typesupport_connext_cpp/service_type_support.h>
 
+inline std::string
+_create_type_name(
+  const message_type_support_callbacks_t * callbacks,
+  const std::string & sep)
+{
+  return std::string(callbacks->package_name) +
+         "::" + sep + "::dds_::" + callbacks->message_name + "_";
```

Comment:
I ran uncrustify over this already and I wouldn't call that indent wrong, but it is an alternative indent that we don't usually use. In this case I think it looks fine, in other cases like this:

```c++
  some_really_long_named_function_that_does_stuff(arg1,
                                                  arg2,
                                                  arg3);
```

I think that style doesn't make sense, but there has to be some room for developer discretion in these types of things.